src/layers/layer-factory.js

```diff
@@ -86,7 +86,8 @@ export const LAYER_VIS_CONFIGS = {
     range: [0, 100],
     step: 0.1,
     group: PROPERTY_GROUPS.stroke,
-    property: 'thickness'
+    property: 'thickness',
+    allowCustomValue: true
```

Comment:
can you add `allowCustomValue: false` to properties that is `type: 'number'`? for example: `opacity`, `coverage`..

Reply:
> can you add `allowCustomValue: false` to properties that is `type: 'number'`? for example: `opacity`, `coverage`..

ok, I also added it to percentage, is it correct?

Reply:
Yes